src/emc/kinematics/genhexkins.c

```diff
@@ -11,7 +11,7 @@
 *    
 * Copyright (c) 2004 All rights reserved.
 *
-* Last change: 2014.12.22
+* Last change: 2016.08.04
```

Comment:
personally I would just delete items like this; git tracks changes in files better than any manual in-file tracking. (copyright years excepted, if you care about 'em, because that can apparently legally matter)